src/tests/BaseCard.test.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from 'redux/store';
+import BaseCurrencyContext from 'context/BaseCurrencyContext';
+import { BrowserRouter as Router } from 'react-router-dom';
+import BaseCard from 'components/BaseCard';
+
+describe('Tests for <BaseCard /> component', () => {
+  const base = (
+    <BaseCard
+      baseCurrency="USD"
+    />
+  );
+  const baseCurrency = 'USD';
+
+  const wrap = (component) => (
+    <Provider store={store}>
+      <BaseCurrencyContext.Provider value={{ baseCurrency }}>
+        <Router>
+          {component}
+        </Router>
```

Comment:
- [ ] Keep doing the great work 👏

Reply:
Hi @iambenkis I have created another video of less than 5 minutes.
loom video:https://www.loom.com/share/8d6957a97e28410d8eb40092b404df68

Thanks